README.md

```diff
@@ -35,6 +35,7 @@ npm i stat-methods
 3.  [Descriptive statistics](#Descriptive-statistics)
     -   [min](#min)
     -   [max](#max)
+    -   [sum](#sum)
```

Comment:
Thanks for this. Would you please also add this to the smaller table of content at below line `306` ? Same for `product` please.